CONTRIBUTING.md

```diff
@@ -1,109 +1,143 @@
 
-```js
-localStorage.setItem('forkEnabled', 'true');
-localStorage.setItem('forkBaseChainId', 1); // the networkId you are forking
-localStorage.setItem('forkNetworkId', '3030'); // the networkId on the fork
-localStorage.setItem('forkRPCUrl', <rpcurl>);
+```bash
+# setting the environment to 'staging' will enable testnet markets, disabling governance, staking, and production markets
+NEXT_PUBLIC_ENV=prod
+
+# you can also disable staking & governance by altering
+NEXT_PUBLIC_ENABLE_GOVERNANCE=true
+NEXT_PUBLIC_ENABLE_STAKING=true
 ```
 
-As localeStorage is not observed you need to **reload** after setting the parameters.
-Now the market selection should show forked markets for all the markets that run on `forkBaseChainId`.
-To do actual transactions on the fork you need to setup your wallet to use the same `rpcurl` you provided as `forkRPCUrl`.
+## Running Against a Chain Fork
 
-## Token addition
+You can run the UI locally against a forked chain network, similar to what the tests do with Tenderly. This will allow you to build and interact with the UI without spending actual funds. This is very useful for testing many protocol scenarios.
 
-To add a new token to the app, all you have to do is adding a svg token icon inside `public/icons/tokens` please make sure the svg name equals the `lowercase` onchain `symbol`.
-If you want a custom name to appear alongside the icon you can alter `src/ui-config/reservePatches.ts` accordingly.
+First, you’ll need to create a fork of a network, which can be done with a few different tools. We suggest using the one created by Bored Ghost Labs, where you can follow the steps and get set up [here](https://github.com/bgd-labs/aave-tenderly-cli).
 
-## Translations
+Second, you’ll need to tell the local application which fork to run against. The easiest way to do this is to copy/paste the following statements. With the application running locally, open up the console in the browser and copy/paste the following with the appropriate values.
+
+```js
+localStorage.setItem('forkEnabled', 'true');
+localStorage.setItem('forkBaseChainId', <chainId>); // the ID for the chain you are forking, in numeric format
+localStorage.setItem('forkNetworkId', '3030'); // the ID of the new forked network
+localStorage.setItem('forkRPCUrl', <rpcUrl>);
+```
 
-Aave uses [Crowdin](https://crowdin.com/project/aave-interface) for translation management.
+Since `localStorage` changes are not not observed, _you’ll need to reload after setting the parameters_. After reloading, the market selection should show forked markets for all the markets that run on `forkBaseChainId`. To make actual transactions on the fork, you’ll need to setup your wallet to use the same `rpcUrl` you provided as `forkRPCUrl`. This will require you to setup your wallet by adding in the new fork network and connecting to the app with it.
```

Comment:
Not sure if we wanna mention about configuring the rpc port in metamask after to point to the tenderly RPC

Reply:
Good catch - I think we may want to tbh - I'll copy over some of the content for that from [here](https://github.com/aave/tenderly-fork)?

Reply:
Added